_rules/SC3-1-1-html-lang-valid.md

```diff
@@ -47,54 +47,102 @@ While HTML5 specification indicates that `xml:lang` attribute takes priority ove
 
 ## Passed
 
+#### Pass example 1
+
+Has valid `lang`.
```

Comment:
This is not a grammatically correct sentence. Neither are the others in this file.